cmake/templates/python_distutils_install.sh.in

```diff
@@ -29,5 +28,5 @@ echo_and_run /usr/bin/env \
     "@CMAKE_CURRENT_SOURCE_DIR@/setup.py" \
     build --build-base "@CMAKE_CURRENT_BINARY_DIR@" \
     install \
-    $DESTDIR_ARG \
+    --root="${DESTDIR-/}" \
```

Comment:
Passing `/` in the non-DESTDIR case appears to be necessary as failing to do so yields:

```
TEST FAILED: home/administrator/catkin/build/install/lib/python3/dist-packages/ does NOT support .pth files
bad install directory or PYTHONPATH
You are attempting to install a package to a directory that is not
on PYTHONPATH and which Python does not read ".pth" files from.  The
installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    home/administrator/catkin/build/install/lib/python3/dist-packages/

and your PYTHONPATH environment variable currently contains:

     '/home/administrator/catkin/build/install/lib/python3/dist-packages:/home/administrator/catkin/build/lib/python3/dist-packages:'
```

Reply:
Seems reasonable. I found another page that references `--root=/` (with `--prefix`) https://docs.brew.sh/Python-for-Formula-Authors#--prefix-vs---root which seems to match what's being done here.